.github/workflows/publish-draft-releases.yml

```diff
@@ -175,98 +175,6 @@ jobs:
-        env:
-          SCCACHE_RELEASE_URL: https://github.com/mozilla/sccache/releases/download
-          SCCACHE_VERSION: v0.2.15
-        run: |
-          SCCACHE_FILE=sccache-$SCCACHE_VERSION-x86_64-unknown-linux-musl
-          mkdir -p $HOME/.local/bin
-          curl -L "$SCCACHE_RELEASE_URL/$SCCACHE_VERSION/$SCCACHE_FILE.tar.gz" | tar xz
-          mv -f $SCCACHE_FILE/sccache $HOME/.local/bin/sccache
-          chmod +x $HOME/.local/bin/sccache
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
-          echo "$HOME/.cargo/bin" >> $GITHUB_PATH
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: cache cargo registry
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-          key: cargo-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            cargo-
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: cache sccache
-        uses: actions/cache@v2
-        continue-on-error: false
-        with:
-          path: /home/runner/.cache/sccache
-          key: sccache-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            sccache-
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: start sccache server
-        run: sccache --start-server
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: init
-        run: |
-          curl -s https://sh.rustup.rs -sSf | sh -s -- -y
-          source ${HOME}/.cargo/env
-          rustup toolchain install stable
-          rustup toolchain install nightly
-          rustup default stable
-          rustup target add wasm32-unknown-unknown --toolchain nightly
-          cargo +nightly install --git https://github.com/alexcrichton/wasm-gc --force
-          rustup update
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: build
-        env:
-          RUST_BACKTRACE: full
-          RUSTC_WRAPPER: sccache
-          SCCACHE_CACHE_SIZE: 2G
-          SCCACHE_DIR: /home/runner/.cache/sccache
-        run: |
-          source ${HOME}/.cargo/env
-          cargo build --verbose --release
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: stop sccache server
-        run: sccache --stop-server || true
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: strip
-        run: |
-          strip target/release/manta
-      -
-        if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
-        name: upload
-        uses: actions/upload-artifact@v2
-        with:
-          name: manta-base
-          path: target/release/manta
```

Comment:
Can you explain why dropping this section? (It is not obvious to me)

Reply:
Ah yes let me also modify the description a bit in that case, as this is the main point of the PR.
We used to test the runtime upgrade by starting the network with old client and old runtime and upgrading the runtime.
This means if there's a breaking client change (which sometimes Parity does) the test will automatically fail.
That's why the network should just start with new client. And so we don't need this secondary build from the `manta` branch.